### PR TITLE
Tailwind CSS

### PR DESCRIPTION
**Pull request by Amit366**
Sir please review the tailwind CSS applied to the navbar

### CODE REVIEW

**Comment by Amit366** [comment]
Sir I am the participant of all the three contests.

**Comment by Amit366** [comment]
crossWOC, MWOC, DWOC

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -45,23 +45,37 @@ function Header() {
...
+                    </li>
+                    <li className="sm:inline-block">
                     <NavLink className={styles.header_links} to='/editor' exact activeClassName={styles.header_active_links}>
                         Editor
                     </NavLink>
+
+                    </li>
+                    <li className="sm:inline-block">
+                    <NavLink className={styles.header_links} to='/sketch' exact activeClassName={styles.header_active_links}>
+                        Sketch <sup><span style={{color: "red", fontSize: "0.7rem"}}>New</span></sup>
```
Comment (ashuvssut) [inline src/components/Header/Header.js:67]: @Amit366  remove the code in line 66 and 67. that code is causing the build error.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -45,23 +45,37 @@ function Header() {
     return (
         <>
             <header className={styles.Header}>
+              <div className="container mx-auto">
+               <div className="sm:flex justify-around">
                 <img className={styles.left} src={logo} alt="Logo"/>
-                <div className={styles.right}>
-                    <Link to='home_banner' offset={-15}>
+                  <ul className="text-gray-700 sm:self-center text-xl border-t sm:border-none">
```
Comment (smaranjitghose) [inline src/components/Header/Header.js:51]: Avoid using inline css as much as possible

**Pull request closed without merging**